src/main/jbake/content/blog/2017/01/13/events-aggregation-extension.adoc

```diff
@@ -0,0 +1,167 @@
+= Extending Complex Event Processing in Hawkular Alerting
+Lucas Ponce
+2017-01-13
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, alerts, complex, events, processing, stream, polling
+
+== Stream Processing versus Polling Processing
+
+Hawkular Alerting uses different techniques to detect behaviours by defining rules.
+
+The main implementation is based in a Stream Processing design. +
+Hawkular Alerting analyzes streams of data and events ordered in a timeline searching matches against conditions. +
+This technique is also known as Event Processing (or Complex Event Processing)specially when the processing of incoming data might need multiple conditions.
+
+This method is particularity good to identify meaningful happenings on complex scenarios and respond to them as quickly as possible.
+
+Hawkular Alerting also supports Polling techniques based on Alerters that allow to define queries that are processing
+against a backend in a periodically manner.
+
+This Polling technique is also good in non real time scenarios, for example, to compare historical values in
+large range time intervals using statistical operators like averages, medians or percentiles.
+
+== Sliding Windows on Complex Event Processing
+
+An interesting aspect in the logic of detection behaviours is when we can combine characteristics of Stream Processing and Polling Processing.
+
+A Sliding Window is a way to define a scope in the stream of data and events received in the timeline. +
```

Comment:
Same issue with line break. Same applies to other paras below, so if fixed here should be fixed in other places.